akshay/HTML/assing5.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>assing5</title>
+        <style>
+            body{
+
+                margin-left: 70px; 
+                background-color: #e6ffff;             
+            }
+        </style>
+    </head>
+    <body>
+    <center>
+        <h1>Google</h1>
+    </center>  
+    <p>This article is about the company.</p> 
+    <h2>From the garage to the Googleplex</h2> 
+    <p>The Google story begins in 1995 at <strong> anford University.<br>
+        Larry Page</strong>was considering Stanford for grad school and <strong> Sergey Brin </strong>,  a student<br>
+        there, was assigned to show him around.</p>
+        <p>
+            By some accounts, they disagreed about nearly everything during that first meeting, but by the following<br>
+            year they struck a <span style = "background-color: rgb(226, 224, 75);"> partnership.</span> <br>
+            Working from their dorm rooms, they built a search engine that used links to determine the importance of<br>
+            individual pages on the <span style="color:blue;text-size-adjust: 20px;"> World Wide Web .</span> <br><br>
+            They called this search engine <span style = "background-color: rgb(226, 224, 75);"> Backrub.</span><br>
+        </p>
+        <hr>
+        <h2>Initial public offering</h2>
+        <p>
+            Google's initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 30 20<br>
```

Comment:
20 should be underlined and use strike tag for 30.

Reply:
In line number 27 Backrub should be italic

Reply:
> In line number 27 Backrub should be italic

done